### PR TITLE
build(tags): update tag to v6 and tag group to v4

### PR DESCRIPTION
**Pull request by jawinn**
## Description

- Use spectrum CSS tag version 6.
- Use spectrum CSS tag group version 4.
- Adjust spectrum config to change pluralized class names that have been renamed to singular.

## Related issue(s)

[Spectrum CSS PR 2039](https://github.com/adobe/spectrum-css/pull/2039)

## How has this been tested?

-   [ ] Test that tags is still rendering correctly on docs page
-   [ ] Test that tags is still rendering correctly in Storybook
-   [ ] Look over VRTs for any possible issues

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [x] Breaking change (fix or feature that would cause existing functionality to change)
-   [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## Checklist

-   [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
-   [x] My code follows the code style of this project.
-   [ ] If my change required a change to the documentation, I have updated the documentation in this pull request.
-   [x] I have read the **[CONTRIBUTING](<(https://github.com/adobe/spectrum-web-components/blob/main/CONTRIBUTING.md)>)** document.
-   [ ] I have added tests to cover my changes.
-   [ ] All new and existing tests passed.
-   [ ] I have reviewed at the Accessibility Practices for this feature, see: [Aria Practices](https://www.w3.org/TR/wai-aria-practices/)

## Best practices

This repository uses conventional commit syntax for each commit message; note that the GitHub UI does not use this by default so be cautious when accepting suggested changes. Avoid the "Update branch" button on the pull request and opt instead for rebasing your branch against `main`.

### CODE REVIEW

**Review by Westbrook — COMMENTED** [review]
This seems like it would be good to go. It need to be caught up to `main` and have `yarn` run locally to ensure all of the compiled CSS is up-to-date.

**Comment by jawinn** [comment]
> This seems like it would be good to go. It need to be caught up to `main` and have `yarn` run locally to ensure all of the compiled CSS is up-to-date.

Thanks! _Tag group_ is on its merry way with being migrated and has a beta, so I was planning to combine that update with this PR, since they both affect the same component and are somewhat connected.

File: packages/overlay/overlay-trigger.md
Comment (jawinn) [inline packages/overlay/overlay-trigger.md:1]: This extra space appeared after merging `main` into my branch for some reason.

**Review by najikahalsema — APPROVED** [review]
rebase to main and we should be good to go!

**Pull request merged**